README.md

```diff
@@ -1,28 +1,32 @@
 # LSP-clangd
+
 C++ support for Sublime's LSP plugin provided through clangd.
```

Comment:
Unrelated, but somehow related,
Should it also mention that it supports C as well?

Reply:
Good point. It does support Objective-C/C++ and CUDA (currently missing in selectors) files as well.

Reply:
Lets keep CUDA support for a separate PR, if someone cares. That would require figuring out syntax to recommend, scope and such.